client/app/components/Input/__tests__/InputCheckboxGroup.spec.jsx

```diff
@@ -10,7 +10,8 @@ const label = 'Some Label';
 const idTwo = 'some-other-id';
 const nameTwo = 'some-other-name';
 const labelTwo = 'Some Other Label';
-const someEvent = () => {
+const someEvent = (hasError) => {
```

Comment:
Minor fix: previously, `window.alert` would be invoked below even if there was no error. The assertion on line 55 helped catch this issue.